src/main/java/com/edu/User.java

```diff
@@ -0,0 +1,39 @@
+package com.edu;
+
+public class User {
+
+    public static final int COM_NUMBER_LENGTH = 3;
+
+    public void validUserNumber(String userNumber) throws IllegalArgumentException {
+        int myNumLength = userNumber.length();
+        // 길이 확인
+        if (myNumLength != COM_NUMBER_LENGTH) {
+            throw new IllegalArgumentException();
+        }
+
+        for (int i = 0; i < myNumLength; i++) {
+            // 숫자가 아닌 경우
+            checkNumber(userNumber, i);
+            // 중복 확인
+            checkOverlapNumber(userNumber, myNumLength, i);
+        }
+
+    }
+
+    private void checkNumber(String userNumber, int i) {
+        char[] ch = userNumber.toCharArray();
+
+        if (ch[i] < 49 || ch[i] > 57) {
+            throw new IllegalArgumentException();
+        }
+    }
+
+    private void checkOverlapNumber(String userNumber, int myNumLength, int i) {
+        for (int j = 0; j < myNumLength; j++) {
+
+            if (i != j && userNumber.charAt(i) == userNumber.charAt(j)) {
+                throw new IllegalArgumentException();
```

Comment:
```suggestion
                throw new IllegalArgumentException("예외처리 메시지");
```

message를 파라미터로 전달하면 예외가 발생했을 때, 해당 메시지를 띄울 수 있습니다. 
어떤 예외를 발생시킨 것인지, 메시지를 입력해주시면 더 명확하게 예외처리를 할 수 있을 것 같습니다.